packages/ui/lib/components/QueuePopup/index.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { withState, withHandlers, compose } from 'recompose';
+import { Icon, Dropdown, Popup } from 'semantic-ui-react';
+
+import Img from 'react-image';
+import styles from './styles.scss';
+
+import artPlaceholder from '../../../resources/media/art_placeholder.png';
+
+export const QueuePopup = ({
+  trigger,
+  dropdownOptions,
+  idLabel,
+  isOpen,
+  handleClose,
+  handleOpen,
+  handleTriggerClick,
+  handleRerollTrack,
+  handleChangeStream,
+  selectedStream,
+  target,
+  titleLabel,
+  track
+}) => {
+  if (!!track.local || !(track.streams && selectedStream)) {
+    return trigger;
+  }
+
+  return (
+    <Popup
+      className={styles.queue_popup}
+      trigger={<div onClick={handleTriggerClick}>{trigger}</div>}
+      open={isOpen}
+      onClose={handleClose}
+      onOpen={handleOpen}
+      hideOnScroll
+      position='right center'
```

Comment:
shouldn't this be left center?

Reply:
when i put "left center", it works fine on story book but fail on the app :/
when "right center", its the opposite :scream: :scream: :scream:

Reply:
Lol, this is horrible. I'll need to replace these popups with something better at some point, but let's stick to these for the time being.

Reply:
I think the problem come from the app, on the storybook the popup act as intended without the target hack.
Maybe a css conflict ?